src/diffusers/models/embeddings.py

```diff
@@ -1038,11 +1090,11 @@ def forward(self, timestep, class_labels, hidden_dtype=None):
 
 
 class CombinedTimestepTextProjEmbeddings(nn.Module):
```

Comment:
we can revert changes to this embedding now, I think it is not used anymore (you replaced it with CogView3CombinedTimestepConditionEmbeddings)